### DISCUSSION TITLE
Use native RDP login only

### DISCUSSION THREAD

**Discussion by VSuc** [neutrinolabs/xrdp#2585] [Q&A]
Hello everyone,

Is it possible to completely disable the xrdp login screen and only rely on native RDP auth?
I need the xrdp server login to behave the same as on Windows, ie:
Either show me my desktop (if login successful) or login prompt like this:
![image](https://user-images.githubusercontent.com/60236074/223732687-65a7b1b4-b591-4641-ad6f-737c836398fc.png)

I tried to set "require_credentials=true", but that's not exactly what i'm trying to achieve, since i still get the "login failed for display ..." instead RDP client prompt.

Thanks a lot

**Comment by matt335672** [comment]
You can get skip the login screen by embedding your username and (encrypted) password in the .ini file.

The pre-auth requires NLA which we haven't yet implemented (see #256). It's a complex protocol and while I can see us doing the Kerberos parts of it in the future I can't see there's much point in the NTLM support.
